### PR TITLE
Run a raiden node documentation. (TASK-53200)

### PR DESCRIPTION
**Pull request by wynstrol**

### CODE REVIEW

File: site/src/pages/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Link from "@docusaurus/Link";
 import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
 import useBaseUrl from "@docusaurus/useBaseUrl";
 import styles from "./styles.module.css";
-import FAQs from "../components/fAQs/faqs";
+import FAQs from "../components/FAQs/faqs";
```
Comment (DenhamPreen) [inline site/src/pages/index.js:8]: Good spot. For some reason this doesn't give issues on my system.

File: site/docs/doc2.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
 ---
 id: doc2
-title: Document Number 2
+title: Run your own Raiden Node
 ---
 
-This is a link to [another document.](doc3.md) This is a link to an [external page.](http://www.example.com/)
+## Setup Node
```
Comment (DenhamPreen) [inline site/docs/doc2.md:6]: @wynstrol Is it possible to add the link to the repo, the clone instructions, etc. I think its best to write this as if you were giving instructions to another developer that had 'fokol' idea of what even a node was, or why it's even needed

**Review by DenhamPreen — CHANGES_REQUESTED** [review]
Great stuff! could you please make it a bit more directed at more novice devs, or somewhat a muggle 🧙🏼‍♂️.

**Comment by wynstrol** [comment]
Yeah sure thing. Will get on it either after later today or first thing tomorrow morning.

**Pull request merged**